packages/hyperion-autologging/src/ALSurfaceUtils.ts

```diff
@@ -0,0 +1,11 @@
+import { AUTO_LOGGING_SURFACE } from "./ALSurfaceConsts";
```

Comment:
hmm, this was just a default value of a configuration flag. how many places do we use the functions in this module?

Reply:
These functions used in one place on line 111 in ALUIEventPublisher